### ISSUE TITLE
Watchtower Shoutrrr Pushbullet error

### ISSUE THREAD

**Issue by michelotten** [containrrr/shoutrrr#212]
Tried setting up pushbullet notifications in watchtower via the shoutrrr service, but running into error messages.
Herewith the docker-compose file and the error messages.

Hope you guys can help me out.
**Docker compose**
```
  Watchtower:
    container_name: Watchtower
    image: containrrr/watchtower:latest
    restart: unless-stopped
    network_mode: none
    command: --cleanup --schedule "0 0 4 * * 6" NodeRED DSMR Zigbee2MQTT MariaDB Portainer Mosquitto Postgres UnifiController Watchtower
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    environment:
      - TZ=Europe/Amsterdam
      - WATCHTOWER_NOTIFICATIONS=shoutrrr
      - WATCHTOWER_NOTIFICATION_URL=pushbullet://o.JhUpnNVV01amPXtzZE9hIR4y89lfnNcs
```
_For safety purposes I removed the pushbullet API key within Pushbullet before putting this message online._

**Error message**
```
goroutine 24 [running]:,
github.com/containrrr/shoutrrr/pkg/services/pushbullet.doSend(0xc0003cca50, 0x0, 0x0, 0xc0000a8b00, 0x15c, 0xc00018a068, 0x0, 0x0),
	/home/runner/go/pkg/mod/github.com/containrrr/shoutrrr@v0.4.4/pkg/services/pushbullet/pushbullet.go:69 +0x328,
github.com/containrrr/shoutrrr/pkg/services/pushbullet.(*Service).Send(0xc0003bc620, 0xc0000a8b00, 0x15c, 0xc00018a068, 0x0, 0x0),
	/home/runner/go/pkg/mod/github.com/containrrr/shoutrrr@v0.4.4/pkg/services/pushbullet/pushbullet.go:35 +0xa5,
github.com/containrrr/shoutrrr/pkg/router.sendToService.func1(0xc0001b4720, 0xdfdd40, 0xc0003bc620, 0xc0000a8b00, 0x15c, 0xc00018a068),
	/home/runner/go/pkg/mod/github.com/containrrr/shoutrrr@v0.4.4/pkg/router/router.go:115 +0x4f,
created by github.com/containrrr/shoutrrr/pkg/router.sendToService,
	/home/runner/go/pkg/mod/github.com/containrrr/shoutrrr@v0.4.4/pkg/router/router.go:115 +0x1a5,
time="2021-11-13T14:27:50+01:00" level=info msg="Watchtower 1.3.0\nUsing notifications: pushbullet\nOnly checking containers with name \"NodeRED\" or \"DSMR\" or \"Zigbee2MQTT\" or \"MariaDB\" or \"Portainer\" or \"Mosquitto\" or \"Postgres\" or \"UnifiController\" or \"Watchtower\"\nScheduling first run: 2021-11-20 04:00:00 +0100 CET\nNote that the first check will be performed in 157 hours, 32 minutes, 9 seconds",
panic: runtime error: invalid memory address or nil pointer dereference,
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xb03c28],
```

**Comment by piksel** [comment]
Seems correct to me, maybe it was a temporary service problem? Watchtower shouldn't crash when that happens of course, and I have fixed the error handling in 0.5 so that it correctly handles responses being `nil`.
If it keeps happening, you can try using `containrrr/watchtower:latest-dev`, which should at least give you the correct error (but it's not a good idea to rely on it, since it's unstable).

**Comment by michelotten** [comment]
Checked again with the dev version and noticed that Watchtower couldn't connect to Postgres. After I've added watchtower container to my host network (previously not linked to a network), notifications started working!
